### PR TITLE
Tweak message step

### LINKED ISSUE #177: Long messages cannot be sent on Slack and Discord
There is a Block API limit:

https://api.slack.com/reference/block-kit/blocks#section_fields

> The text for the block, in the form of a text object. Maximum length for the text in this field is 3000 characters. This field is not required if a valid array of fields objects is provided instead.

We have some errors:

> Slack returned error code "invalid_blocks"

We must add a limit.

### PR DESCRIPTION
**Pull request by pyrech**
Fix #177 and #178

See commit messages for a changelog

### CODE REVIEW

File: templates/santa/message.html.twig [hunk 1]
```diff
@@ -5,12 +5,18 @@
 {% block santa_content %}
     <div class="run-block">
         <form method="post" id="run_form" action="{{ path('message', { application: application }) }}">
+            {# Prevent implicit submission of the form - no need for JS here (see https://stackoverflow.com/a/51507806/1917092) #}
+            <button type="submit" disabled style="display: none" aria-hidden="true"></button>
```
Comment (damienalexandre) [inline templates/santa/message.html.twig:9]: TIL! This is very nice :+1:

File: templates/santa/message.html.twig [hunk 2]
```diff
@@ -121,6 +129,13 @@
         }
       }, false);
 
+      function updateMessageLength() {
+        messageLength.innerHTML = message.value.length;
```
Comment (damienalexandre) [inline templates/santa/message.html.twig:133]: We may have a difference between the PHP count and this count, specialy with emoji involved.

Maybe we should count code-point and not graphemes.

- PHP: `strlen`
- JS: `[...str].length`

See https://stackoverflow.com/a/54369605/665923

**Review by damienalexandre — COMMENTED** [review]
Awesome fix :+1: thanks

File: templates/santa/message.html.twig (hunk 2 above)
Comment (pyrech) [reply templates/santa/message.html.twig:133]: Done

**Pull request merged**